sync-timeoffs/SyncTimeOffs.js

```diff
@@ -911,7 +917,7 @@ function convertOutOfOfficeToTimeOff_(timeOffTypeConfig, employee, event, existi
         endAt: endAt,
         typeId: timeOffType.attributes.id,
         typeName: timeOffType.attributes.name,
-        comment: event.summary.replace(' [synced]', ''),
+        comment: event.summary.replace(' ⇵', ''),
```

Comment:
Why not check for the exact char here? I know we always add a whitespace but checking for a whitespace on removal but not when adding it (we use `includes` there) seems error prone.

I would suggest removing the whitespace from the replace, worst case we chain some whitespaces in event summaries.

Reply:
Your suggestion makes sense.

However, I would advise against generally passing a modified string with a white-space added on the end to Personio.

Thus, I opted to make the symbol match more specific and include the optional space character in the match and removal steps.

Testing done in the browser:
![grafik](https://github.com/giantswarm/gapps-automation/assets/18163495/71e836be-11d4-4c0c-9c16-98d003c54217)